content/java/concepts/calendar/terms/internalGet/internalGet.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: '.internalGet()'
+Description: 'Returns the value of a given field'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Calendar'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.internalGet()`** method is a protected method used to retrieve the value of a field from a Calendar instance. In other words it takes a field as input and returns its corresponding value. This method is generally used within the Calendar class or its sub classes.
+
+## Syntax
+
+```pseudo
+protected final int internalGet(int field)
+```
+
```

Comment:
There should be a note here after the syntax regarding the parameters taken, return value, or any other valuable information